### PR TITLE
Add recipe for nb_black

### PR DESCRIPTION
**Pull request by xhochy**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/staged-recipes This recipe is ready for review/merge.

File: recipes/nb_black/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - black
```
Comment (chrisburr) [inline recipes/nb_black/meta.yaml]: I know it’s implicit from the other two but it’s probably nicer to include `python` explicitly:
```suggestion
    - python
    - black
```

File: recipes/nb_black/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (chrisburr) [inline recipes/nb_black/meta.yaml]: This needs to be made Python 3.6+ only or use selectors as it has different dependencies on 2.7: https://github.com/dnanhkhoa/nb_black/blob/master/setup.py#L28-L29

Comment (xhochy) [reply recipes/nb_black/meta.yaml]: As we explicitly depend on `black`, this recipe should already be Python 3.6+ but I can make this more visible in the `run` requirements.

**Pull request merged**